aws/modules/k8s/Rakefile

```diff
@@ -21,7 +21,7 @@ rule "#{KOPS_OUT_DIR}/kubernetes.tf" do
   # Create cluster iff it doesn't already exist.
   sh "kops get cluster #{ENV["TF_VAR_cluster_name"]}" do |ok, res|
     unless ok
-      node_size = ENV["TF_VAR_cluster_name"].start_with?("stg.","prd.") ? "c4.xlarge" : "t2.medium"
+      node_size = ENV["TF_VAR_cluster_name"].start_with?("stg.","prd.") ? "t2.medium" : "t2.medium"
```

Comment:
Why not just write it as the following?

```
    node_size = "t2.medium"
```

This would optimize for readability.

Reply:
My thinking was that we would need to re-add this code the next time we change node_size.

But this is a fair point. YAGNI.